.yamllint

```diff
@@ -0,0 +1,38 @@
+---
+# Based on ansible-lint config
+
+extends: default
+
+rules:
```

Comment:
> When YAML is used as the format of a communication channel, it is useful to be able to indicate the end of a [document](https://yaml.org/spec/1.1/#document/syntax) without closing the [stream](https://yaml.org/spec/1.1/#stream/syntax), independent of starting the next [document](https://yaml.org/spec/1.1/#document/syntax). Lacking such a marker, the YAML [processor](https://yaml.org/spec/1.1/#processor/) reading the [stream](https://yaml.org/spec/1.1/#stream/syntax) would be forced to wait for the header of the next [document](https://yaml.org/spec/1.1/#document/syntax) (that may be long time in coming) in order to detect the end of the previous one. To support this scenario, a YAML [document](https://yaml.org/spec/1.1/#document/syntax) may be terminated by an explicit end line denoted by “...”, followed by optional [comments](https://yaml.org/spec/1.1/#comment/syntax). To ease the task of concatenating YAML [streams](https://yaml.org/spec/1.1/#stream/syntax), the end marker may be repeated.
[Source](https://yaml.org/spec/1.1/#document%20boundary%20marker/)

WDYT? Do we want to end our yamls like that or not?

Reply:
@DamianBudelewski what is the benefit for us in using this? Ansible playbooks do not support multiple YAML documents within the same file, so according to their docs the document-start/end is [optional](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics).

Reply:
For me, it just looked better. We don't need to do it